Sources/Views/MusicProgressSliderCell.m

```diff
@@ -0,0 +1,83 @@
+//
+//  MusicProgressSlider.m
+//  Hermes
+//
+//  Created by Xinhong LIU on 19/4/15.
+//
+//
+
+#import "MusicProgressSliderCell.h"
+
+@implementation MusicProgressSliderCell
+
+- (NSRect)knobRectFlipped:(BOOL)flipped {
+  CGPoint sliderOrigin = [self barRectFlipped:false].origin;
+  CGSize knobSize = CGSizeMake(2.0, 8.0); // this value is measured in iTunes App
+  CGSize sliderSize = [self barRectFlipped:false].size;
+  
+  CGPoint knobOrigin;
+  knobOrigin.x = truncf(sliderOrigin.x + [self progressInPercentage]
+                        * (sliderSize.width - knobSize.width));
+  // truncf is important to make knob's border clear and sharp.
```

Comment:
Move this comment above one line, usually comments about code go above or to the side, never below.